docs/assets/javascripts/binding-phrase.js

```diff
@@ -0,0 +1,42 @@
+function getBytesFromWordArray(wordArray) {
+  const result = [];
+
+  result.push(wordArray.words[0] >>> 24);
+  result.push((wordArray.words[0] >>> 16) & 0xff);
+  result.push((wordArray.words[0] >>> 8) & 0xff);
+  result.push(wordArray.words[0] & 0xff);
+
+  result.push(wordArray.words[1] >>> 24);
+  result.push((wordArray.words[1] >>> 16) & 0xff);
+
+  return result;
+}
+
+function uidBytesFromText(text) {
+  const bindingPhraseFull = `-DMY_BINDING_PHRASE="${text}"`;
+  const bindingPhraseFullEncoded = CryptoJS.enc.Utf8.parse(bindingPhraseFull);
+
+  const bindingPhraseHashed = CryptoJS.MD5(bindingPhraseFullEncoded);
+  const uidBytes = getBytesFromWordArray(bindingPhraseHashed);
+
+  return uidBytes;
+}
+
+function setOutput(text) {
+  const uidBytes = uidBytesFromText(text);
+  const output = document.querySelector("#__code_0 code");
```

Comment:
Is it going to break when additional code blocks are added to this page?

Reply:
I have redone the element selection to only look for empty code blocks and to fail silently if they are not found.